xc-tool/build.rs

```diff
@@ -0,0 +1,107 @@
+use std::{env, fs, path::PathBuf, path::Path};
+
+use ethers::contract::Abigen;
+use ethers::solc::{remappings::Remapping, Artifact, Project, ProjectPathsConfig, Solc};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let pyth_evm_src_root =
+        env::var("PYTH_XC_TOOL_EVM_SRC_ROOT").unwrap_or("../ethereum/".to_string());
+    let evm_path = PathBuf::from(&pyth_evm_src_root).canonicalize()?;
+
+    let solc = Solc::default();
+    eprintln!("Got Solc!");
```

Comment:
nit, again for later: using a structured logging library such as https://crates.io/crates/structlog will make things easier as the project grows

Reply:
I can't put my finger on it, but I've never clicked with the structured logging pattern. You grep for them anyway in the end. But I know it's very little extra hassle so I think we should do it if you think it's a good fit here.